di_website/common/blocks.py

```diff
@@ -3,28 +3,49 @@
 from django.core.exceptions import ValidationError
 from django.forms import Media
 from django.forms.utils import ErrorList
-
-from wagtail.core.blocks import (
-    BooleanBlock,
-    CharBlock,
-    ChoiceBlock,
-    ListBlock,
-    PageChooserBlock,
-    RichTextBlock,
-    StreamBlock,
-    StructBlock,
-    TextBlock,
-    URLBlock
-)
+from wagtail.core.blocks import (BooleanBlock, CharBlock, ChoiceBlock,
+                                 ListBlock, PageChooserBlock, RichTextBlock,
+                                 StreamBlock, StructBlock, TextBlock, URLBlock)
 from wagtail.documents.blocks import DocumentChooserBlock
 from wagtail.embeds.blocks import EmbedBlock
 from wagtail.images.blocks import ImageChooserBlock
 
-from di_website.common.constants import RICHTEXT_FEATURES, RICHTEXT_FEATURES_NO_FOOTNOTES
+from di_website.common.constants import (RICHTEXT_FEATURES,
+                                         RICHTEXT_FEATURES_NO_FOOTNOTES)
 from di_website.publications.blocks import AudioMediaBlock
 from di_website.publications.infographic import Infographic
 
 
+RED = 'poppy'
+BLUE = 'bluebell'
+PINK = 'rose'
+YELLOW = 'sunflower'
+ORANGE = 'marigold'
+PURPLE = 'lavendar'
+GREEN = 'leaf'
+COLOUR_CHOICES = (
+    (RED, 'Red'),
+    (BLUE, 'Blue'),
+    (PINK, 'Pink'),
+    (YELLOW, 'Yellow'),
+    (ORANGE, 'Orange'),
+    (PURPLE, 'Purple'),
+    (GREEN, 'Green')
+)
```

Comment:
This is redundant as it already exists (almost identically) in the publications modal page. This make sense since this kind of theming is only applicable to publication (resource) pages, and not all of the e.g AudioVisual don't support it. See my next comment for how to handle theming for these different types of pages.